lua/lspconfig/server_configurations/twig_language_server.lua

```diff
@@ -0,0 +1,26 @@
+local util = require 'lspconfig.util'
+
+return {
+  default_config = {
+    cmd = { 'twig-language-server', '--stdio' },
+    filetypes = { 'twig' },
+    root_dir = function(pattern)
+      local cwd = vim.loop.cwd()
+      local root = util.root_pattern('composer.json', '.git')(pattern)
```

Comment:
just `return util.root_pattern('composer.json')` and enalbe single_file_mode.

Reply:
Okay, I made the changes.